README.rst

```diff
@@ -156,6 +156,48 @@ To change default order use:
     local t = require('luatest')
     t.configure({shuffle = 'group'})
 
+---------------------------------
+Preloaded hooks
+---------------------------------
+Preloaded hooks extend base hooks. They behave like the pytest fixture with the ``autouse`` parameter.
+
+.. code-block:: lua
+    -- my_helper.lua
+    local hooks = require('luatest.hooks')
+
+    hooks.before_all_preloaded(function() print('foo') end)
+
+    hooks.before_all_preloaded(function() print('bar') end)
+    hooks.after_all_preloaded(function() print('baz') end)
+
+    hooks.after_all_preloaded(function() print('qux') end)
+
+If you run the following test:
+
+.. code-block:: lua
+    local t = require('luatest')
+    local my_helper = require('my_helper')
+    local g = t.group()
+
+    g.before_all(function() print('prepare') end)
+    g.after_all(function() print('cleanup') end)
+
+    g.test_foo = function() print('everythings is ok') end
+
+Then the hooks are executed in the following sequence:
+
+.. code-block:: text
+    |\ before_suite
+    | \ before_all_preloaded (foo)
+    |  \ before_all_preloaded (bar)
+    |   \ before_all (prepare)
+    |    \
+    |      test_foo (everythings is ok)
+    |    /
+    |   / after_all_preloaded (baz)
+    |  / after_all_preloaded (qux)
+    | / after_all (cleanup)
```

Comment:
Shouldn't `after_all (cleanup)` be right after `test_foo`?

Reply:
Fixed.